helm/alfresco-content-services/templates/deployment-filestore.yaml

```diff
@@ -13,6 +13,8 @@ metadata:
 spec:
   replicas: {{ .Values.filestore.replicaCount }}
   template:
+    annotations:
```

Comment:
shouldn't this be under metadata?

Reply:
Thanks for spotting it.  I will take a look into it.

Reply:
no worries, I fixed in #212 which is now merged